compose-builder/get-api-gateway-token.sh

```diff
@@ -24,16 +24,32 @@ if [ "$DEV" = "-dev" ]; then
   CORE_EDGEX_VERSION=0.0.0
 fi
 
-rm -rf keys
-mkdir keys
-openssl ecparam -name prime256v1 -genkey -noout -out keys/ec256.key 2> /dev/null
-cat keys/ec256.key | openssl ec -out keys/ec256.pub 2> /dev/null
+# Init key dir
+GW_KEY_DIR=/tmp/edgex-gateway-keys
+
+rm -rf ${GW_KEY_DIR}
+mkdir -p ${GW_KEY_DIR}
+
+# Build Gateway Key
+openssl ecparam -name prime256v1 -genkey -noout -out ${GW_KEY_DIR}/gateway.key 2> /dev/null
+openssl ec -in ${GW_KEY_DIR}/gateway.key -pubout -out ${GW_KEY_DIR}/gateway.pub 2> /dev/null
+
+# JWT File
+JWT_FILE=/tmp/edgex/secrets/edgex-security-proxy-setup/kong-admin-jwt
+JWT_VOLUME=/tmp/edgex/secrets/edgex-security-proxy-setup
+#JWT=`cat ${JWT_FILE}`
 
 ID=`uuidgen`
```

Comment:
This can be removed as it is already assigned value and exported below. I forget to remove when I did mine.

Reply:
remove ID=`uuidgen` the whole line

Reply:
We still need ID=uuidgen - that is the secret material that makes up the inner portion of the JWT. I have taken out the export line though.